src/apps/media/src/app/components/MediaDetailsModal.js

```diff
@@ -73,37 +75,42 @@ export const MediaDetailsModal = memo(function MediaDetailsModal(props) {
         </div>
 
         <div className={styles.FieldsContainer}>
-          <CopyButton
-            className={styles.CopyButton}
-            kind="outlined"
-            value={props.file.url}
-          />
-
           <FieldTypeText
-            className={styles.Field}
-            name="title"
-            value={title}
+            className={styles.InputCopyCombo}
+            name="filename"
+            value={filename}
             label={
               <label>
-                <Infotip title="Edit title. | Use for alt text with Parsley's .getImageTitle()" />
-                &nbsp;Title
+                <Infotip
+                  className={styles.InfotipFileName}
+                  title="Edit Filename to update your image URL path "
+                />
+                &nbsp;Edit Filename URL path
```

Comment:
"Edit Filename URL path" feels to overdone. I would suggest we leave it as just "URL Filename". This accomplishes the following;
- removes the verb of the action being done "Edit". Not present on other fields.
- removes the description "path" as we do not support multi slug path parts which this seems to indicate.